build/cmake/Config.cmake.in

```diff
@@ -0,0 +1,8 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(ZLIB)
+find_dependency(Libevent)
```

Comment:
Same, we need `CONFIG`:
* https://docs.hunter.sh/en/latest/packages/pkg/Libevent.html